admin/syslog-ng/files/syslog-ng.conf

```diff
@@ -1,8 +1,5 @@
-#############################################################################
-# OpenWrt syslog-ng.conf specific file
-# which collects all local logs into a single file called /var/log/messages.
-# More details about these settings can be found here:
-# https://www.syslog-ng.com/technical-documents/list/syslog-ng-open-source-edition
+# Collect all local logs into a single file /var/log/messages.
+# See https://www.syslog-ng.com/technical-documents/list/syslog-ng-open-source-edition
```

Comment:
It seems that this link goes only to version 3.38 and we do have already version 4.4. :-(

Reply:
It loooks like they didn't updated yet docs. Anyway, after the change the stats dumping remains disabled but no any warnings.